harvester_e2e_tests/integration/test_networks.py

```diff
@@ -0,0 +1,840 @@
+
+
+import subprocess
+
+from time import sleep
+from datetime import datetime, timedelta
+
+import pytest
+import paramiko
+import shlex
+
+
+pytest_plugins = [
+    "harvester_e2e_tests.fixtures.api_client",
+    'harvester_e2e_tests.fixtures.network',
+    'harvester_e2e_tests.fixtures.vm',
+    "harvester_e2e_tests.fixtures.virtualmachines"
+]
+
+
+image_name = "suse"
+vm_credential = {"user": "opensuse", "password": "123456"}
+node_user = "rancher"
+node_password = "p@ssword"
```

Comment:
I added the node password in `host-password` of config.yaml. 
Use the configure option to retrieve the password for all test cases
  ```
  node_password = request.config.getoption("--host-password")
  ```
And verified can pass all test cases on dolphin cluster 

Thanks for the reminder.

Reply:
Could you please update the PR (push commits) before update comments?
Otherwise It would always confusing me that received the mail but code is not changed.

I would suggest you can just update PR and click *Re-request review*, then I can check the newer changed directly and mark it resolved or add more comments in further. (it would also reduce mails flooding my mail box)